server/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const { MongoClient } = require('mongodb')
+const path = require('path')
+const bodyParser = require('body-parser')
+const CardsRouter = require('./routes/cards-router.js')
```

Comment:
Unless dealing with `class`es, stick with `lowerCamelCase` for your function and variable names.